frontend/src/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Banana</title>
+  </head>
+  <body>
+    <div id="root"></div>
```

Comment:
```suggestion
```

Now that you actually changed `index.tsx` to suit my taste, which I totally appreciate, this `div` is no longer required.